docs/prerequisites.rst

```diff
@@ -3,7 +3,7 @@ System Prerequisites
 
 The following software packages are required to be installed on your system:
 
-* `Python 2.7 or 3.4+ <https://www.python.org>`_
+* `Python 2.7 or 3.5+ <https://www.python.org>`_
```

Comment:
Should line 22 be changed to "work on **newer** versions of Python 3 as well"?

Reply:
Done.